presto-native-execution/src/test/java/com/facebook/presto/spark/TestPrestoSparkNativeSimpleQueries.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spark;
+
+import com.facebook.presto.testing.ExpectedQueryRunner;
+import com.facebook.presto.testing.QueryRunner;
+
+/**
+ * Following JVM argument is needed to run Spark native tests.
```

Comment:
Should we move this somewhere? There will be multiple tests and copy-pasting this into each test would be less than ideal.

Reply:
Yeah maybe I can move it to the common parent abstract class AbstractTestPrestoSparkQueryFramework

Reply:
Got it. I assume we don't need to split the test into abstract class and test class though. Do we?

Reply:
Yeah we don't for this one.

Reply:
Actually the previous implementation is wrong. We cannot have that common parent for Spark. I can't think of a more proper single place to put these documentation other than the PrestoSparkNativeQueryRunnerUtils class. (That somehow doesn't feel a good place either, but at least we get a single place to refer to.)